.github/ISSUE_TEMPLATE/project.md

```diff
@@ -21,35 +21,57 @@ assignees: ''
 
 <!-- Please fill out the following questions to rationalize the prioritization of this project -->
 
-- Does this serve our mission?
+- Does this serve our mission
 
 - Will this help to make us financially sustainable within a year?
 
-- Is it feasible to build and maintain with our team and runway?
-
 - Can this make us move faster?
 
-- Does this increase quality and not technical debt?
+- Does this increase quality? 
+
+- Does this add technical debt?
+
+- Is it feasible to build and maintain with our current team and runway?
 
-- Will this bring us more Collective and financial contributors?
+- What insight or research is this project based on?
+- Is thisproblem already solved elsewhere?
 
-- Is this already solved elsewhere or is it our unique contribution?
+- How will we measure our success? Do we have a baseline today that we can compare this against?
 
-## Roadmap & Timeframe
-(Key milestones or phases and estimated delivery dates, including specs, design, feature development, feedback, marketing/comms, and documentation)
+- How well do we understand the complexity of the problem, feature to be added or user need to be addressed? 
 
-**Kick-off meeting**:
+- What is the estimated timeframe for delivering this project? 
 
-## Success metrics
+## Kick-off meeting
+
+### Roadmap & Timeframe
+(Key milestones or phases and estimated dates. Consider including further scoping, discovery, and prototyping. Include as a minimum: planning, design, development, documentationm, feedback, iteration and, marketing )
+
+### Success metrics
 (What we will judge outcomes of the project by)
 
+### Release planning
+
+** Estimated delivery date**
+
+- Which aspect(s) of the platform does this change impact?
+- Which group(s) of users will benefit from this change?
+- Is this project in response to a user-requested feautre? If so who requested it (link to feature request if available)?
+- Is this project related to any similar work we have done, or are planning on doing in the next three months?
+
+### Next steps
+
+- [ ] Open up any appropriate project issues
+- [ ] Open a documentation issue, link it to this one
+- [ ] Open a release issue, link it to this one
+- [ ] Create a project board, add this issue to the description, add any issues created above 
+
 ## Project Closure
 
 - Close all relevant project issues
 - Open up issues for improvements 
+- Post comm's according to comm's plan (most likely in release issue)
 - Post project screencast on documentation slack channel 
-- Are comms for the project needed?
+- Post comm's according to comm's plan
```

Comment:
Remove this line I guess since it's already written as the third bullet point. 🤔 

```suggestion

```